molecule/common/verify.yml

```diff
@@ -0,0 +1,59 @@
+---
+- name: Start the QDR
+  hosts: qdrtest
+  tasks:
+    - name: Check for qdrouterd.conf
+      find:
+        paths: "{{ qdr_conf_output_dir }}"
+        patterns: qdrouterd.conf
+      register: conf
+      failed_when: conf.files|length != 1
+
+    - name: Start qdrouterd
+      command: qdrouterd -U qdrouterd -d
+
+- name: Functional tests
+  hosts: localhost
+  tasks:
+    - block:
+      - name: Clear the failure flag
+        set_fact: functest_failed_flag="unknown"
+
+      - name: Run the qdr test from STF functional-tests
+        include_role:
+          name: functional_tests
+          tasks_from: test_qdr
+        vars:
+          qdr_container_name: 'qdrtest'
```

Comment:
container_bin should be set here to use docker.

Reply:
I see it's set below with the host_var in molecule.yml

Reply:
As long as the env var CONTAINER_BIN is set, 
vars:
    container_bin: "{{ lookup('env', 'CONTAINER_BIN') }}"
    
 should work